src/controllers/__tests__/authenticationController.test.ts

```diff
@@ -998,6 +1046,175 @@ describe("AuthenticationController|>LV|>acs", () => {
   );
 });
 
+describe("AuthenticationController|>LV|>acs|>notify user login", () => {
+  const setupLollipopScenario = () => {
+    jest
+      .spyOn(authCtrl, "isUserElegibleForFastLogin")
+      .mockImplementationOnce((_) => true);
```

Comment:
We can use this instead.
```suggestion
      .mockReturnValueOnce(true);
```